proposals/0184-dismiss-alert.md

```diff
@@ -1,4 +1,4 @@
-# Dismiss Alert RPC
+# Cancel Interaction RPC
 
 * Proposal: [SDL-0184](0184-dismiss-alert.md)
```

Comment:
Do you want to change this file name to cancel-interaction?

Reply:
Fixed